examples/chat-13B.bat

```diff
@@ -0,0 +1,50 @@
+@echo off
+
+cd ..
+if not "%errorlevel%"=="0" (
+    echo Unable to move up in the directory structure.
+    pause
+    exit /b 1
+)
+
+if not defined MODEL set "MODEL=models\13B\ggml-model-q4_0.bin"
+if not defined USER_NAME set "USER_NAME=User"
+if not defined AI_NAME set "AI_NAME=ChatLLaMa"
+rem Adjust to the number of CPU cores you want to use.
+if not defined N_THREAD set "N_THREAD=8"
```

Comment:
```suggestion
rem if not defined N_THREAD set "N_THREAD=8"
```

Reply:
a) The main code already has default value of for threads set to max threads available in local machine. Makes no sense to circumvent this and set threads to 8 if user has 2/4/6 threads on their CPU. I see that you're just using the same values as the linux script, but the linux script is wrong and should be changed. Left commented, it's still easy for anyone to change if wanted to.